TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/bot/BotActivity.java

```diff
@@ -189,7 +196,7 @@ public void onError(String message) {
 
     @Override
     public void moveIn(Direction direction) {
-        debugView.showPermanently(direction.visualRepresentation());
+        debugView.showTimed(direction.visualRepresentation());
```

Comment:
To match that of the human otherwise we won't know when events are no longer being emitted.